src/utils/filtersInterface.ts

```diff
@@ -4,6 +4,7 @@ export interface Filter {
   id: string;
   cardLabel: string;
   filterLabel: string;
+  category?: string;
```

Comment:
Where is this used?

Reply:
Nice catch. I had considered using this during development but then forgot to remove it